src/test/run-pass/signal-exit-status.rs

```diff
@@ -0,0 +1,29 @@
+// copyright 2013 the rust project developers. see the copyright
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/copyright.
+//
+// licensed under the apache license, version 2.0 <license-apache or
+// http://www.apache.org/licenses/license-2.0> or the mit license
+// <license-mit or http://opensource.org/licenses/mit>, at your
+// option. this file may not be copied, modified, or distributed
+// except according to those terms.
+
+// xfail-fast
+
+use std::{os, run};
+use std::rt::io::process;
+
+fn main() {
+    let args = os::args();
+    if args.len() >= 2 && args[1] == ~"signal" {
+        // Raise a segfault.
+        unsafe { *(0 as *mut int) = 0; }
```

Comment:
This is UB and has remained unfixed on master, the address should be `1` or something else, not NULL.
Might work on windows too then.